be/Controller/RemoveGameCtrl.php

```diff
@@ -62,30 +61,29 @@ private function removeLastGame(
 
     private function removeLastGameIfPossibleInDb(string $code): array
     {
-        $user = $this->userRepository->requireUserByCode($code);
-
-        $userNid = $user->getUserNid();
-
         try {
-            $lastGame = $this->gameRepository->requireLastGame($userNid);
+            $user = $this->userRepository->requireUserByCode($code);
         } catch (NoResultException $e) {
             return [
                 'removed' => false,
-                'warning' => 'Player has no games',
+                'warning' => 'Player does not exist',
             ];
         }
 
-        $opponentUser = $this->getOpponentFromGame($lastGame, $userNid);
+        $userNid = $user->getUserNid();
 
         try {
-            $opponentLastGame = $this->gameRepository->requireLastGame($opponentUser->getUserNid());
```

Comment:
Ten przypadek nie miał szansy zaistnieć bo jeśli wyżej sprawdzamy czy user ma jakieś gry i z tej gry pobieramy oponenta to on musi mieć też gry